VERSION.md

```diff
@@ -2,7 +2,8 @@ RemoteDialer follows a pre-release (v0.x) strategy of semver. There is limited c
 
 The current supported release lines are:
 
-| RemoteDialer Branch | RemoteDialer Minor version | 
+| RemoteDialer Branch | RemoteDialer Minor version | Matching Rancher Version |
```

Comment:
Do we need to maintain this info here? remotedialer does not exist only for rancher.

Reply:
I wondered why it wasn't there, but isn't it a useful guideline for people using remotedialer with rancher?

Reply:
No. People may use this with **wrangler** but not with **rancher**, so there is no need to maintain rancher version info here.

See the dependency graph for other projects that are using this module - many of them without wrangler or rancher: https://github.com/rancher/remotedialer/network/dependents